SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/SpatialRPCService.cpp

```diff
@@ -352,7 +352,7 @@ EPushRPCResult SpatialRPCService::PushRPCInternal(const Worker_EntityId EntityId
 	const uint64 NewRPCId = RPCStore.LastSentRPCIds.FindRef(EntityType) + 1;
 
 	// Check capacity.
-	if (LastAckedRPCId + RPCRingBufferUtils::GetRingBufferSize(Type) >= NewRPCId)
+	if (RPCRingBufferUtils::ShouldIgnoreCapacity(Type) || LastAckedRPCId + RPCRingBufferUtils::GetRingBufferSize(Type) >= NewRPCId)
```

Comment:
Why introduce should ignore capacity when we could treat this the same way as multicast and assume that all movements are acked? Or maybe we should move multicast to always ignore capacity rather than assume its last RPC was "acked"

Reply:
There is a subtle difference in that when the capacity is explicitly ignored, it's valid for the receiving side to miss the RPCs that were overwritten. While we don't guarantee that this won't happen for multicast, it may still be valuable to warn when that happens since it could indicate there are too many multicast RPCs being sent.